index.html

```diff
@@ -1612,45 +1633,49 @@ <h3 id="applying">
           <code>dir</code> member
```

Comment:
Can you replace this with: `<dfn>dir</dfn>member`, and the same for the rest?

Reply:
you won't need `<code>`, ReSpec will automatically code them.

Reply:
I actually think it was done this way so that you dont have cursive in the headers, which looks stupid. The next line does the dfn